devs/docs/tests.rst

```diff
@@ -10,6 +10,24 @@ Run the all of `doctests`_::
 
     $ ./gradlew itest
 
+For running itest on Windows WSL needs to be installed. If downloaded Linux distro comes without Java, install it by running the following commands::
```

Comment:
For properly having links within reStructureText, use `<reference>`_. Also, please use a maximum line length of 80 characters.

```suggestion
For running itest on Windows, `WSL needs to be installed`_. If downloaded Linux
distro comes without Java, install it by running the following commands::
```

Reply:
Thanks for pointing it out, good catch.